packages/salesforcedx-utils-vscode/test/jest/context/workspaceContextUtil.test.ts

```diff
@@ -7,7 +7,14 @@
 
 import { AuthInfo, Connection } from '@salesforce/core';
 import * as vscode from 'vscode';
-import {  WorkspaceContextUtil } from '../../../src';
+import { ConfigAggregatorProvider, WorkspaceContextUtil } from '../../../src';
```

Comment:
Pulled in config aggregator in order to mock the call to it for unit testing this file.  Could have mocked the whole module as well.